### PR TITLE
LRAC-11902, LRAC-12266, LRAC-12267

### PR DESCRIPTION
**Pull request by ilzamcmed**
[https://issues.liferay.com/browse/LRAC-11902](https://issues.liferay.com/browse/LRAC-11902)
[https://issues.liferay.com/browse/LRAC-12266](https://issues.liferay.com/browse/LRAC-12266)
[https://issues.liferay.com/browse/LRAC-12267](https://issues.liferay.com/browse/LRAC-12267)

### CODE REVIEW

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ViniLopes20** [comment]
ci:test:relevant

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/attributes/AccountsAttributesModal.tsx [hunk 1]
```diff
@@ -14,36 +14,43 @@
...
+		label: Liferay.Language.get('data-type'),
+		value: 'dataType',
+	},
+	{
+		expanded: true,
+		label: Liferay.Language.get('sample-data'),
+		value: 'sampleData',
+	},
+];
+
```
Comment (interaminense) [inline modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/attributes/AccountsAttributesModal.tsx:38]: Hi @ilzamcmed, instead create a column for accounts, please move it from `PeopleAttributesModal` to inside this component https://github.com/liferay/liferay-portal/blob/master/modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/attributes/Modal.tsx and all components that call the Modal should use the same column.

**Review by interaminense — CHANGES_REQUESTED** [review]

**Review by fmarcio — APPROVED** [review]
Hey @ilzamcmed thanks for working on that!
Once you update the PR with @interaminense changes request, LGTM!

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/attributes/AccountsAttributesModal.tsx (hunk 1 above)
Comment (ilzamcmed) [reply modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/attributes/AccountsAttributesModal.tsx]: Fixed

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ilzamcmed** [comment]
ci:test:sf

**Review by interaminense — APPROVED** [review]
LGTM! Thanks @ilzamcmed

**Review by fmarcio — APPROVED** [review]
LGTM! :rocket:

**Comment by interaminense** [comment]
ci:forward

**Pull request closed without merging**